src/snowflake/cli/_plugins/git/manager.py

```diff
@@ -32,7 +32,7 @@ class GitStagePathParts(StagePathParts):
     def __init__(self, stage_path: str):
         self.stage = GitManager.get_stage_from_path(stage_path)
         stage_path_parts = Path(stage_path).parts
-        git_repo_name = stage_path_parts[0].split(".")[-1]
+        git_repo_name = stage_path_parts[0].split(".")[-1].lower()
```

Comment:
is it case insensitive?

Reply:
Yes, since git_repo_name is a snowflake identifier and they are case insensitive